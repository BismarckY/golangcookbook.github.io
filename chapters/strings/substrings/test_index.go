package main

import "fmt"

func main() {
	s := "Hello World"
	fmt.Println(s[:5])                // the first 5 characters
	fmt.Println(s[:len(s)-6])         // all of the characters except the last 6
	fmt.Println(s[len(s)-5 : len(s)]) // the last 5 characters
	fmt.Println(s[1 : len(s)-1])      // all except the first and last
}
