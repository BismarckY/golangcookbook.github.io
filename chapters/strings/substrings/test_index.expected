Hello
Hello
World
ello Worl
